desktop/components/main_layout/header/view.coffee

```diff
@@ -97,7 +97,7 @@ module.exports = class HeaderView extends Backbone.View
     false
 
   openAuth: (options) ->
-    if options.mode is 'signup' and sd.NEW_ARTIST_PAGE_CTA is 'new_cta' and sd.ON_ARTIST_PAGE
+    if options.mode is 'signup' and sd.ARTIST_PAGE_CTA_ENABLED
```

Comment:
In this case, if you click 'Sign up' in the header and you're on the artist page with the new CTA, we want to just trigger the takeover.